etl/steps/data/garden/explorers/2021/food_explorer.ipynb

```diff
@@ -1,2290 +1,2530 @@
+    "    ).country.nunique()\n",
+    "    x = x.assign(\n",
+    "        population=population_,\n",
+    "    )\n",
+    "\n",
+    "    # add real population\n",
+    "    population_ = population.reset_index().astype({\"year\": float})\n",
+    "    x = x.merge(\n",
+    "        population_, left_on=[\"region\", \"year\"], right_on=[\"country\", \"year\"]\n",
+    "    ).rename(columns={\"population_y\": \"population_gt\", \"population_x\": \"population\"})\n",
+    "    if ncountries:\n",
+    "        # add real number of countries\n",
+    "        region_size = []\n",
+    "        for r, members in regions.items():\n",
+    "            region_size.append({\"region\": r, \"ncountries_gt\": len(members)})\n",
+    "        r = pd.DataFrame(region_size)\n",
+    "        x = x.merge(r, left_on=\"region\", right_on=\"region\")\n",
+    "    # build df\n",
+    "    x = pd.DataFrame(x)\n",
+    "    # diff population\n",
+    "    x = x.assign(\n",
+    "        population_diff=x.population_gt - x.population,\n",
+    "        population_diff_perc=(x.population_gt - x.population) / x.population_gt,\n",
+    "    )\n",
+    "    return x"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "711580c4-d34c-426f-b49d-56b45defd0ba",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# continents\n",
+    "x_cont = build_df(x.assign(region=x.country.map(country2continent)))\n",
+    "# income groups\n",
+    "x_inco = build_df(x.assign(region=x.country.map(country2income)))\n",
+    "# world\n",
+    "x_world = build_df(x.assign(region=\"World\"), ncountries=False)\n",
+    "# merge\n",
+    "x = pd.concat([x_cont, x_inco, x_world], ignore_index=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "df7b9205-241d-4d7b-a97c-45af5b6dfcd9",
+   "metadata": {},
+   "source": [
+    "Time to filter out these samples based on our heuristic (t1, t2) and go back to `fe_bulk`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "01d32430-73ef-4a9e-80f0-fc3216ce21e4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Tresholds\n",
+    "t1 = 0.1\n",
+    "t2 = 0.2\n",
+    "# Metrics to apply filter\n",
+    "metrics = [\n",
+    "    \"food_available_for_consumption__fat_g_per_day\",\n",
+    "    \"food_available_for_consumption__kcal_per_day\",\n",
+    "    \"food_available_for_consumption__kg_per_year\",\n",
+    "    \"food_available_for_consumption__protein_g_per_day\",\n",
+    "    \"other_uses__tonnes\",\n",
+    "    \"waste_in_supply_chain__tonnes\",\n",
+    "    \"feed__tonnes\",\n",
+    "]\n",
+    "# get removable\n",
+    "remove_all = x.loc[\n",
+    "    (x.population_diff_perc >= t2) & (x.metric.isin(metrics)),\n",
+    "    [\"product\", \"metric\", \"region\", \"year\"],\n",
+    "]\n",
+    "remove_capita = x.loc[\n",
+    "    (x.population_diff_perc >= t2)\n",
```

Comment:
This line should be dropped right? Otherwise we disregard t1 entirely and only ever remove per-capita values if `>= t2`!

Currently, Oceania still has per-capita metrics in total.csv (wrongly).

Reply:
Checked it out, with that line removed it works fine!

Reply:
Well spotted! I'm changing it to `x.population_diff_perc < t2`